src/App.tsx

```diff
@@ -1,5 +1,7 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect } from 'react';
 import classNames from 'classnames';
+import * as postsActions from './features/posts';
```

Comment:
Ensure that the actions imported from './features/posts' are correctly defined and exported in the posts module.